code/style.css

```diff
@@ -1,3 +1,178 @@
 body {
-  background: #FFECE9;
-}
\ No newline at end of file
+  background: #f8e2cf;
+  display: flex;
+  flex-direction: column;
+  color: black;
+  font-family: "Montserrat", sans-serif;
+}
+
+#user-info {
+  display: flex;
+  justify-content: center;
+}
+
+.profile-img {
+  border-radius: 50%;
+  width: 30%;
+  height: 30%;
+  align-self: center;
+  box-shadow: 8px 8px 8px rgb(114, 114, 114);
+}
+
+#profile-name {
+  flex-direction: column;
+  margin-left: 20px;
+}
+
+header {
+  display: flex;
+  justify-content: center;
+}
+
+.github-icon {
+  width: 20%;
+  height: auto;
+}
+
+h1 {
+  align-self: center;
+  font-size: 25px;
+}
+
+h2 {
+  font-size: 20px;
+}
+
+h3 {
+  font-size: 12px;
+}
+
+#projects {
+  display: grid;
+  grid-template-columns: repeat(auto-fit, minmax(300px, 1fr));
+  column-gap: 1rem;
+  row-gap: 1rem;
+  margin: 30px 0px 20px 0px;
+}
+
+.project-box {
+  background: #ff5c58;
+  border-bottom-left-radius: 10px;
+  border-bottom-right-radius: 10px;
+  padding: 10px 0px 10px 20px;
+  box-shadow: 10px 10px 5px rgb(155, 155, 155);
+  -moz-box-shadow: 10px 10px 5px rgb(155, 155, 155);
+  -webkit-box-shadow: 10px 10px 5px rgb(155, 155, 155);
+  -khtml-box-shadow: 10px 10px 5px rgb(155, 155, 155);
+}
+
+.project-box:hover {
+  background-color: #bf3d39;
+  -webkit-transition: 0.3s ease-in-out;
+  transition: 0.3s ease-in-out;
+  -moz-box-shadow: none;
+  -webkit-box-shadow: none;
+  box-shadow: none;
+}
+
+a {
+  font-size: 15px;
+  color: black;
+  text-decoration: none;
+  text-transform: capitalize;
+}
+
+.usr-name:hover {
+  color: #ff5c58;
+  text-transform: lowercase;
```

Comment:
This lowercase effect was mind blowing for me! It's such a delicate and nice touch!!!